### DISCUSSION TITLE
Deployment for a client and service provider

### DISCUSSION THREAD

**Discussion by AbedAltalhi** [Security-Onion-Solutions/securityonion#9593] [Unsupported Versions]
I am looking for the best choice for the following scenario,
I am in a company and we want to monitor a client company network. what is the best deployment for that, and if you can provide installation guide that will be so helpful.

**Comment by robbiemarshall** [comment]
You would want a distributed installation with a forward node at the client, and your manager and/or search node at your location.  Starting here in the documentation should get you going:  https://docs.securityonion.net/en/2.3/getting-started.html

**Reply by AbedAltalhi** [reply]
What about if I have multiple clients? I can use multiple  forwarder nodes with one  manager and/or search node?

**Reply by robbiemarshall** [reply]
Yes, you can have multiple forward nodes with one manager.  The number of search nodes will depend on how many forward nodes you deploy.

**Reply by tiger-greenhell** [reply]
Attention, Security Onion is not a multitenant solution.
